recipe/meta.yaml

```diff
@@ -1,29 +1,33 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
-# Using the name variable with the URL in line 13 is convenient
-# when copying and pasting from another recipe, but not really needed.
 {% set name = "vega-cli" %}
-{% set version = "5.6.0" %}
+{% set version = "5.17.0" %}
+{% set sha256 = "38aadfe34a45f4065cafb2eef9232d69331a331f2847af2293ef682580147478" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
+source:
+  url: https://github.com/vega/vega/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
 build:
   number: 0
-  skip: true  # [win]
+  noarch: generic
```

Comment:
This is isn't possible with NodeJS packages as the executable paths are different on Windows.

Reply:
I suspected as much